develop/reference/articles/01-intro/00-development-reference-intro.markdown

```diff
@@ -1,113 +1,61 @@
+Liferay CE artifact repository<span class="opens-new-window-accessible"> (Opens New Window)</span></a>.
 </p>
 
 <h3><span style="font-size: 22px;">Java APIs</span></h3>
 
 <p style="margin-left: 40px;">
-A module's Javadoc describes its Java API. To find specific API modules, you can
-refer to <a
-href="/develop/reference/-/knowledge_base/7-0/finding-liferay-api-modules">
-Finding Liferay API Modules</a>.
-</p>
-
-
-<p style="margin-left: 40px;">
-You can download a newly migrated module's Javadoc
-JAR file from <a
-href="https://repository.liferay.com/nexus/content/repositories/liferay-releases-ce/com/liferay/"
-target="_blank">Liferay Nexus repository<span
-class="opens-new-window-accessible"> (Opens New Window)</span></a> and extract
-it to a local folder. The extracted <code>index.html</code> file is the gateway
-to exploring the module's Java API.
-</p>
-
-<p style="margin-left: 40px;">
-Here are the Java packages common to newly migrated API modules:
-<ul style="margin-left: 60px;">
-<li>
-<code>com.liferay.[component].constants</code> - Classes that specify module
-specific constant values, such as web keys
-</li>
-<li>
-<code>com.liferay.[component].exception</code> - Module specific exception classes
-</li>
-<li>
-<code>com.liferay.[component].model</code> - Model entity interfaces, entity wrapper classes, and SOAP classes
-</li>
-<li>
-<code>com.liferay.[component].service</code> - Local and remote service interfaces
-</li>
-<li>
-<code>com.liferay.[component].service.persistence</code> - Entity persistence interfaces, entity finder interfaces, and static utilities
-</li>
-<li>
-<code>com.liferay.[component].util</code> - Utility classes
-</li>
-</ul>
-</p>
-
-<p style="margin-left: 40px;">
-Not all components have been migrated from the <a
-href="https://docs.liferay.com/portal/7.0/javadocs/portal-kernel/" 
-target="_blank">portal-kernel API<span
-class="opens-new-window-accessible"> (Opens New Window)</span></a> (formerly named portal-service). Their API
-modules have proxies to classes in portal-kernel. Their Javadoc is available at
-<a href="https://docs.liferay.com/portal/7.0/javadocs/portal-kernel/"  
-target="_blank">https://docs.liferay.com/portal/<span
+<span style="font-size:18px;">
+<a href="https://docs.liferay.com/portal/7.0/javadocs/" target="_blank">
+Portal API&nbsp;<span class="opens-new-window-accessible">(Opens New Window)</span>
+</a>
+</span>
+</p>
+
+<p style="margin-left: 40px;">
+Describes Liferay Portal 7.0 CE modules and standalone app modules at the time
+of the latest GA release. You can also download them from the 
+<a href="https://www.liferay.com/downloads" target="_blank">Downloads page<span
 class="opens-new-window-accessible"> (Opens New Window)</span></a>.
 </p>
 
+<p style="margin-left: 40px;">&nbsp;</p>
+
+<p style="margin-left: 40px;">
+<span style="font-size:18px;">
+<a href="https://docs.liferay.com/portal/7.0/javadocs/" target="_blank">
```

Comment:
Hey Jim,

Sure, but what's the alternative? if there are so many modules then not listing them would make it even harder for a dev to find them.

I think that having many can be solved by organizing them in groups and using tables so that they fit in an smaller area.